todolist/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.urls import path, re_path
+
+from .views import *
+
+urlpatterns = [
+    path('', index, name='home'),
+    path('about/', about, name='about'),
+    path('create/', add_snippet, name='create'),
+    path('delete/', delete_snippet, name='delete'),
+    path('login/', login, name='login'),
+    path('post/<int:post_id>/', show_post, name='post'),
+    path('category/<int:cat_id>/', show_category, name='category')
```

Comment:
Can you explain what these endpoints do?
The mist interesting for me is `create`, `delete`, `post` and `category.`
The reason why I'm asking is CRUD. If you have entity and you want to create/read/update/delete it you can use the same endpoint for all of this operations. The only difference can be HTTP verbs (POST/GET/PUT/DELETE).